packages/clay-css/src/scss/atlas/variables/_sidebar.scss

```diff
@@ -60,7 +60,10 @@ $sidebar-light: map-deep-merge(
 	(
 		background-color: $white,
 		border-left-width: 0px,
-		box-shadow: -0.25rem 0 0.5rem -0.25rem rgba(0, 0, 0, 0.1),
+		box-shadow: 0.25rem 0 0.5rem -0.25rem rgba(0, 0, 0, 0.1),
+		sidenav-right: (
+			box-shadow: -0.25rem 0 0.5rem -0.25rem rgba(0, 0, 0, 0.1),
+		),
```

Comment:
@pat270 maybe the default couldn't be the box-shadow property of `sidenav-right`? It seems that it's more common to have a sidebar on the right side so we don't have to go out and update these components in DXP as well.

I think we do not use `sidenav-right` correctly everywhere in DXP.

Reply:
@matuzalemsteles Good point. We will need to add `sidenav-left` or `sidenav-start` in this case.

Reply:
We might as well use `sidenav-left` since we already have `sidenav-right`, but we also have another nomenclature like `c-slideout-start` for VerticalBar. I don't know if we can use it or is it a separate class for another purpose.

Reply:
I can add aliases so we will have `sidenav-left`, `sidenav-start`, `sidenav-right`, `sidenav-end`.